packages/typespec-test/test/batch_modular/generated/typespec-ts/src/api/pagingHelpers.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
```

Comment:
if the plan is to diverge from core and re-align, maybe we should consider moving these helpers into the package temporarily so we don't have to cast between two similar types?